test/Orchard.Tests/Extensions/TestDependencyModules/DerivedThemeSample2/Theme.txt

```diff
@@ -0,0 +1,5 @@
+﻿Name: Derived Theme Sample
+Author: Nicholas Mayne
+Description: Base Theme
+Version: 1.0.0 
+BaseThemes: BaseThemeSample, BaseThemeSample2
```

Comment:
BaseTheme and dependency are two things. A theme can depend on other features, and can inherit from one theme, the implied behavior is not the same.